api4/file.go

```diff
@@ -565,7 +576,11 @@ func getFileLink(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	resp := make(map[string]string)
-	resp["link"] = c.App.GeneratePublicLink(c.GetSiteURLHeader(), info)
+	link := c.App.GeneratePublicLink(c.GetSiteURLHeader(), info)
+	resp["link"] = link
+
+	auditRec.Success()
+	auditRec.AddMeta("link", link)
```

Comment:
Would it make sense to always leave the `auditRec.Success()` as the last thing (that is, to always `AddMeta` before marking the record as successful) for consistency?

Reply:
I can see how that makes sense, but the convention thus far has been to set the success flag then add the final meta data, which is usually the modified data.  If you feel strongly about it I can it, and am happy to make the change, but it will need to be changed in about 150 places.

Reply:
If we have a convention in place already I'm happy to leave it as is